extensions/wikia/PageHeader/templates/PageHeaderController_categories.php

```diff
@@ -0,0 +1,29 @@
+<?php if ( $categories->hasVisibleCategories() ): ?>
+	<div class="page-header__categories">
+		<span class="page-header__categories-in" data-tracking="categories-top-in"><?= $categories->inCategoriesText ?>:</span>
+		<div class="page-header__categories-links">
+			<?= $categories->visibleCategoriesHTML ?>
```

Comment:
In my opinion it is better to return html from controller than being depended on code formatting in template, we avoid bugs that could appear after using autoformatter ;)